doc/api/n-api.md

```diff
@@ -515,6 +515,8 @@ the currently running Agent which was set by means of a previous call to
 `napi_set_instance_data()` will be overwritten. If a `finalize_cb` was provided
 by the previous call, it will not be called.
 
+Not compatible with `libnode`.
```

Comment:
What would this mean?

Reply:
That API call is not to be used when using `libnode`

Reply:
Why not?

Reply:
Yes, but that doesn’t mean that this call is not compatible with libnode or with embedded mode.